gnomad_qc/v5/resources/basics.py

```diff
@@ -187,6 +187,15 @@ def get_aou_vds(
     n_samples_after = vds.variant_data.count_cols()
     logger.info("Removed %d samples from VDS.", n_samples_before - n_samples_after)
 
+    # Remove the chr4 site with excessive numbers of alleles (n=22233) which
+    # might cause memory issue later on
```

Comment:
```suggestion
    # Remove the chr4 site with excessive numbers of alleles (n=22233) to
    # avoid memory issues with `split_multi`.
```
copying the format of https://github.com/broadinstitute/gnomad_qc/blob/d4159eba1bc3e0ba99f38953925329ebe411f810/gnomad_qc/v4/resources/basics.py#L188